main/src/cgeo/geocaching/enumerations/LogType.java

```diff
@@ -15,158 +15,158 @@
+    DIDNT_FIND_IT(3, "3", "didn't find it", "Didn't find it", R.string.log_dnf, R.drawable.mark_red, R.drawable.marker_not_found_offline),
+    NOTE(4, "4", "write note", "Comment", R.string.log_note, R.drawable.mark_gray, R.drawable.marker_note),
+    PUBLISH_LISTING(1003, "24", "publish listing", "", R.string.log_published, R.drawable.mark_green_more),
+    ENABLE_LISTING(23, "23", "enable listing", "Ready to search", R.string.log_enabled, R.drawable.mark_green_more),
+    ARCHIVE(5, "5", "archive", "Archived", R.string.log_archived, R.drawable.mark_red_more, R.drawable.marker_archive),
+    UNARCHIVE(12, "12", "unarchive", "", R.string.log_unarchived, R.drawable.mark_green_more),
+    TEMP_DISABLE_LISTING(22, "22", "temporarily disable listing", "Temporarily unavailable", R.string.log_disabled, R.drawable.mark_red_more),
+    NEEDS_ARCHIVE(7, "7", "needs archived", "", R.string.log_needs_archived, R.drawable.mark_red, R.drawable.marker_archive),
+    WILL_ATTEND(9, "9", "will attend", "Will attend", R.string.log_attend),
+    ATTENDED(10, "10", "attended", "Attended", R.string.log_attended, R.drawable.mark_green, R.drawable.marker_found_offline),
+    RETRIEVED_IT(13, "13", "retrieved it", "", R.string.log_retrieved, R.drawable.mark_green_more),
+    PLACED_IT(14, "14", "placed it", "", R.string.log_placed, R.drawable.mark_green_more),
+    GRABBED_IT(19, "19", "grabbed it", "", R.string.log_grabbed, R.drawable.mark_green_more),
+    NEEDS_MAINTENANCE(45, "45", "needs maintenance", "Needs maintenance", R.string.log_maintenance_needed, R.drawable.mark_red, R.drawable.marker_maintenance),
+    OWNER_MAINTENANCE(46, "46", "owner maintenance", "", R.string.log_maintained, R.drawable.mark_green_more, R.drawable.marker_owner_maintenance),
+    UPDATE_COORDINATES(47, "47", "update coordinates", "Moved", R.string.log_update, R.drawable.marker_owner_maintenance),
+    DISCOVERED_IT(48, "48", "discovered it", "", R.string.log_discovered, R.drawable.mark_green),
+    POST_REVIEWER_NOTE(18, "18", "post reviewer note", "", R.string.log_reviewer),
+    SUBMIT_FOR_REVIEW(76, "76", "submit for review", "", R.string.log_submit_for_review),
+    VISIT(1001, "75", "visit", "", R.string.log_tb_visit, R.drawable.mark_green),
+    WEBCAM_PHOTO_TAKEN(11, "11", "webcam photo taken", "", R.string.log_webcam, R.drawable.mark_green, R.drawable.marker_found_offline),
+    ANNOUNCEMENT(74, "74", "announcement", "", R.string.log_announcement),
+    MOVE_COLLECTION(69, "69", "unused_collection", "", R.string.log_movecollection),
+    MOVE_INVENTORY(70, "70", "unused_inventory", "", R.string.log_moveinventory),
+    RETRACT(25, "25", "retract listing", "", R.string.log_retractlisting),
+    MARKED_MISSING(16, "16", "marked missing", "", R.string.log_marked_missing, R.drawable.mark_red),
+    OC_TEAM_COMMENT(83, null, "X1", "OC Team comment", R.string.log_oc_team_comment),
+    UNKNOWN(0, "unknown", "", "", R.string.err_unknown, R.drawable.mark_red); // LogType
```

Comment:
Hmm… what's with this comment? (I know, it was there already before this commit.)